docs/recipes/ReactNativeVisionCamera.md

```diff
@@ -0,0 +1,414 @@
+---
+title: React Native Vision Camera
+description: How to integrate VisionCamera in Ignite v9+
+tags:
+  - Expo
+  - VisionCamera
+  - react-native-vision-camera
+last_update:
+  author: Frank Calise
+publish_date: 2023-10-23
+---
+
+# VisionCamera
+
+## Overview
+
+VisionCamera is a powerful, high-performance React Native Camera library. It's both feature-rich and flexible! The library provides the necessary hooks and functions to easily integrate camera functionality in your app.
+
+In this example, we'll take a look at wiring up a barcode scanner. This tutorial is written for the Ignite v9 Prebuild workflow, however it generally still applies to DIY or even a bare react-native project.
+
+## Installation
+
+If you haven't already, spin up a new Ignite application:
+
+```terminal
+npx ignite-cli@next new PizzaApp --remove-demo --workflow=prebuild --yes
+cd PizzaApp
+```
+
+Next, let's install the necessary dependencies. You can see complete installation instructions for `react-native-vision-camera` [here](https://react-native-vision-camera.com/docs/guides).
+
+```terminal
+npx expo install react-native-vision-camera
+```
+
+Add the plugin to `app.json` as per the documentation. It'll look like the following if you have the default Ignite template:
+
+```json
+"plugins": [
+  "expo-localization",
+  [
+    "expo-build-properties",
+    {
+      "ios": {
+        "newArchEnabled": false
+      },
+      "android": {
+        "newArchEnabled": false
+      }
+    }
+  ],
+  [
+    "react-native-vision-camera",
+    {
+      "cameraPermissionText": "$(PRODUCT_NAME) needs access to your Camera.",
+      "enableCodeScanner": true
+    }
+  ]
+],
+```
+
+To get this native dependency working in our project, we'll need to run prebuild so Expo can execute the proper native code changes for us. Then we can boot up the app on a device.
+
+```terminal
+npx expo prebuild
+yarn android
```

Comment:
Maybe add a note about why we're using android? Mostly because you can't use the camera on the simulator and it's more difficult to build to an iOS device.